resources/assets/components/Post/index.js

```diff
@@ -60,39 +60,42 @@ class Post extends React.Component {
     const signup = this.props.signup;
     const campaign = this.props.campaign;
     const quantity = post.quantity != null ? post.quantity : 0;
+    const containerSize = post.type === 'photo' ? '-third' : '-half';
 
     return (
       <div className="post container__row">
         {/* Post Images */}
-        <div className="container__block -third images">
-          <div className="post__image">
-            <img src={getImageUrlFromPost(post, 'original')} />
-          </div>
-          <div className="admin-tools">
-            <div className="admin-tools__links">
-              <a href={getImageUrlFromPost(post, 'original')} target="_blank">Original Photo</a>
+        {post.type === 'photo' ?
+          <div className="container__block -third">
+            <div className="post__image">
+              <img src={getImageUrlFromPost(post, 'original')} />
             </div>
-            {this.props.rotate ?
-              <div className="admin-tools__rotate">
-                {this.state.loading ?
-                  <div className="spinner" />
-                  :
-                  <a className="button -tertiary rotate" onClick={this.handleClick} />
-                }
+            <div className="admin-tools">
+              <div className="admin-tools__links">
+                <a href={getImageUrlFromPost(post, 'original')} target="_blank">Original Photo</a>
               </div>
+              {this.props.rotate ?
+                <div className="admin-tools__rotate">
+                  {this.state.loading ?
+                    <div className="spinner" />
+                    :
+                    <a className="button -tertiary rotate" onClick={this.handleClick} />
+                  }
+                </div>
+                : null}
+            </div>
+            {this.props.showSiblings ?
+              <ul className="gallery -duo">
+                {
+                  map(this.getOtherPosts(post), (post, key) => <PostTile key={key} details={post} />)
+                }
+              </ul>
               : null}
           </div>
-          {this.props.showSiblings ?
-            <ul className="gallery -duo">
-              {
-                map(this.getOtherPosts(post), (post, key) => <PostTile key={key} details={post} />)
-              }
-            </ul>
-            : null}
-        </div>
+          : null}
 
         {/* User and Post information */}
-        <div className="container__block -third">
+        <div className={"container__block " + containerSize}>
```

Comment:
Can a template string work here?

`container__block  -${containerSize}`

Reply:
hmm I tried to do that but it didn't like it :-/

Reply:
Oh sorry you need the llittle ticks 
```
`container__block -${containerSize}`
```

Reply:
oh yes - this reads so much better!